components/main_page/wikis/pubg/main_page_layout_data.lua

```diff
@@ -21,21 +49,54 @@ local CONTENT = {
 	},
 	transfers = {
 		heading = 'Transfers',
-		body = '{{Transfer List|limit=15|title=}}\n<div style{{=}}"display:block; text-align:center; padding:0.5em;">\n' ..
-			'<div style{{=}}"display:inline; float:left; font-style:italic;">\'\'[[#Top|Back to top]]\'\'</div>\n' ..
-			'<div style{{=}}"display:inline; float:right;" class="plainlinks smalledit">' ..
-			'&#91;[[Special:EditPage/Player Transfers/{{CURRENTYEAR}}/{{CURRENTMONTHNAME}}|edit]]&#93;</div>\n' ..
-			'<div style{{=}}"white-space:nowrap; display:inline; margin:0 10px font-size:15px; font-style:italic;">' ..
-			'[[Portal:Transfers|See more transfers]]<span style="font-style:normal; padding:0 5px;">&#8226;</span>' ..
-			'[[Special:RunQuery/Transfer history|Transfer query]]' ..
-			'<span style{{=}}"font-style:normal; padding:0 5px;">&#8226;</span>' ..
-			'[[lpcommons:Special:RunQuery/Transfer|Input Form]]' ..
-			'</center></div>\n</div>',
+		body = Fragment{
+			children = {
+				TransferList{ limit = 15 }:fetch():create(),
+				Div{
+					css = { display = 'block', ['text-align'] = 'center', padding = '0.5em' },
+					children = {
+						Div{
+							css = { display = 'inline', float = 'left', ['font-style'] = 'italic' },
+							children = { Link{ children = 'Back to top', link = '#Top'} }
+						},
+						Div{
+							classes = { 'plainlinks', 'smalledit' },
+							css = { display = 'inline', float = 'right' },
+							children = { '&#91;', Link{ children = 'edit', link = getTransferPage() }, '&#93;' },
+						},
+						Div{
+							css = {
+								['white-space'] = 'nowrap',
+								display = 'inline',
+								margin = '0 10px',
+								['font-size'] = '15px',
+								['font-style'] = 'italic'
+							},
+							children = {
+								Link{ children = 'See more transfers', link = 'Portal:Transfers' },
+								CENTER_DOT,
+								Link{ children = 'Transfer query', link = 'Special:RunQuery/Transfer_history' },
+								CENTER_DOT,
+								Link{ children = 'Input Form', link = 'lpcommons:Special:RunQuery/Transfer' }
+							}
+						},
+					}
+				}
+			}
+		},
 		boxid = 1509,
 	},
 	thisDay = {
-		heading = 'This day in PUBG <small id="this-day-date" style = "margin-left: 5px">' ..
-			'({{#time:F}} {{Ordinal|{{#time:j}}}})</small>',
+		heading = Fragment{
+			children = {
+				'This day in PUBG ',
+				Small{
+					attributes = { id = 'this-day-date' },
+					css = { ['margin-left'] = '5px' },
+					children = { '(' .. os.date('%B') .. ' ' .. Ordinal.toOrdinal(tonumber(os.date('%d'))) .. ')' }
+				}
+			}
+		},
 		body = '{{Liquipedia:This day}}',
```

Comment:
could probably make this a widget too (eventually) that replaces the thisDay module